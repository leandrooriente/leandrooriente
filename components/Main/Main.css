.wrapper {
  width: 100%;
  max-width: 960px;
  padding: 0 30px;
  margin: 0 auto;
}