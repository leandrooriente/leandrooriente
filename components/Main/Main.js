import React from 'react';
import PropTypes from 'prop-types';
import css from './Main.css';

const Main = ({ children }) => (
  <section className={css.wrapper}>
    { children }
  </section>
);

Main.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Main;
