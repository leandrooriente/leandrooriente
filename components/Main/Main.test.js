import React from 'react';
import { mount } from 'enzyme';
import Main from './Main';

describe('<Main />', () => {
  it('should render <Header /> component', () => {
    const wrapper = mount(
      <Main>
        <span>Children</span>
      </Main>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
