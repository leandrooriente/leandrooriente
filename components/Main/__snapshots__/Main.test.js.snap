// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render <Header /> component 1`] = `
<Main>
  <section
    className="wrapper"
  />
</Main>
`;
